contributing.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to {{Project-Name}}
```

Comment:
minor: throughout the doc, should we move capitalize words, instead of lowercase (other than prepositions, etc)?